README.md

```diff
@@ -43,10 +43,12 @@ jobs:
   serial: true
   plan:
   - get: resource-web-app
+  - task: build
+    file: resource-web-app/build.yml
   - put: resource-deploy-web-app
     params:
-      manifest: resource-web-app/manifest.yml
-      path: resource-web-app
```

Comment:
It depends on your use case of course; if you want to pass just the binary around then you'd do what this example currently says.